documentation/frontend/reference/testing_guidelines.md

```diff
@@ -357,15 +357,43 @@ output of failed tests by finding the `trace.zip` for the relevant test in the
 [Playwright Trace Viewer](https://playwright.dev/docs/trace-viewer) to inspect
 these (or open the zip yourself and poke around the files on your own).
 
-For Playwright tests failing in CI, a GitHub comment will appear with a link to
-download an artifact of the `test-results` folder.
+Playwright tests in CI are run with `-u` option by default, this means that
+snapshots will automatically be updated for modified parts of the UI if
+Playwright detects that. See
+[Updating snapshots](/frontend/guides/test.md#updating-snapshots) for more
+reading about this.
+
+When this happens, a GitHub comment will appear with a link to download zipped
+artifacts named in the form `*_snapshot_diff.zip`. Once downloaded, they can be
+decompressed and applied to your working branch by running:
+
+`ov unzip -p *_snapshot_diff.zip | git apply`
+
+The above command basically uses the `unzip` tool to unpack the contents of the
+downloaded archive file named `*_snapshot_diff.zip`, the `-p` option prevents it
+from actually creating any extracted file but rather prints the contents to the
+standard output which is then piped to `git apply`.
+
+```{note}
+For the above command to work verbatim for you, make sure to have saved the downloaded artifact to the root of the project from where you're able to run git commands. Otherwise, you'd need to adjust the path to the downloaded file in the command accordingly. Should you settle for the former, keep in mind that you'd also need to delete the file(`*_snapshot_diff.zip`) after successfully applying the changes to avoid committing them.
+
+Remember to replace `*_snapshot_diff.zip` with the actual downloaded filename.
+
+Additionally, you might need to replace `ov` with a relative path `./ov` if you do not have `ov` added to your PATH yet.
```

Comment:
You can leave this out, in favour of the general advice we've written in #4567. We've decided not to have to write this caveat everywhere and just have it in a single place.